example_test.go

```diff
@@ -26,6 +26,14 @@ func example_connect() *tarantool.Connection {
 	return conn
 }
 
+func example_connect_with_opts(opts tarantool.Opts) *tarantool.Connection {
+	conn, err := tarantool.Connect(server, opts)
+	if err != nil {
+		panic("Connection is not established: " + err.Error())
+	}
+	return conn
+}
```

Comment:
It will be more obvious from the outside to add arguments to example_connect():

```Go
func example_connect(server string, opts tarantool.Opts) {
}
```

At now it is not very clear what and where to change at first glance. Up to you.

Reply:
Changed